tests/src/services/users.service.test.ts

```diff
@@ -59,7 +65,7 @@ describe('UsersService tests', () => {
     jest.restoreAllMocks();
   });
 
-  describe('Insert User in Mongo DB', () => {
+  describe('insertUser()', () => {
```

Comment:
This goes directly against what we talked about in the last review. Talk about the use case, not about the function literally